data/src/main/java/com/microsoft/azure/kusto/data/exceptions/KustoServiceQueryError.java

```diff
@@ -33,6 +28,31 @@ public KustoServiceQueryError(String message) {
         this.exceptions.add(new RuntimeException(message));
     }
 
+    public static KustoServiceQueryError fromOneApiErrorArray(ArrayNode jsonExceptions, boolean isOneApi) {
+        List<RuntimeException> exceptions = new ArrayList<>();
+        StringBuilder sb = new StringBuilder();
+
+        if (jsonExceptions == null || jsonExceptions.isEmpty()) {
+            return new KustoServiceQueryError("No exceptions were returned from the service.");
+        }
+
+        if (jsonExceptions.size() > 1) {
+            sb.append(EXCEPTIONS_MESSAGE);
+        }
+
+        for (int i = 0; i < jsonExceptions.size(); i++) {
+            if (isOneApi) {
+                exceptions.add(new DataWebException(jsonExceptions.get(i).toString()));
+            } else {
+                exceptions.add(new RuntimeException(jsonExceptions.get(i).toString()));
+            }
+            sb.append(jsonExceptions.get(i).toString());
+            sb.append("\n");
```

Comment:
did we decide this together with yochai and all ? that we will print them all as message ?
anyway i dont think the message should contain jsonExceptions.get(i).toString(),
maybe a UT is needed so we see the output and know this is what we want